dpnp/dpnp_iface_counting.py

```diff
@@ -79,8 +64,6 @@ def count_nonzero(in_array1, axis=None, *, keepdims=False):
         Parameter ``axis`` is supported only with default value `None`.
         Parameter ``keepdims`` is supported only with default value `False`.
```

Comment:
@densmirn , maybe after this limitation add line like "Otherwise an error will be raised"?